src/toGamut.js

```diff
@@ -104,3 +112,85 @@ export default function toGamut (color, {method = defaults.gamut_mapping, space
 }
 
 toGamut.returns = "color";
+
+/**
+ * toGamutCSS
+ * Given a color `origin`, returns a new color that is in gamut using the CSS
+ * Gamut Mapping Algorithm. If `space` is specified, it will be in gamut in
+ * `space`. Otherwise, it will be in gamut in the color space of `origin`. It
+ * will always be returned in the original color space.
+ * @param {Object} origin 
+ * @param {Object} options 
+ * @param {ColorSpace} options.space 
+ * @returns 
+ */
+export function toGamutCSS(origin, { space = origin.space }) {
+	const JND = 0.02;
+	const ε = 0.0001;
+	const returnSpace = origin.space;
+
+	if (space.isUnbounded) return to(origin, space);
+
+	const origin_OKLCH = to(origin, ColorSpace.get('oklch'));
+	let L = origin_OKLCH.coords[0];
+
+	// return media white or black, if lightness is out of range
+	if (L >= 1) {
+		const white = to(parse('white'), space);
+		white.alpha = origin.alpha;
+		return to(white, returnSpace);
```

Comment:
I want to confirm that this follows the spec's intention- It says `if the Lightness of origin_Oklch is greater than or equal to 100%, return { 1 1 1 origin.alpha } in destination`. I took that to mean "white" in the destination space, for instance `oklab(1 0 0)`, and not `oklab(1 1 1)` or even `oklab(1 .4 .4)`.

Reply:
This was my interpretation as well: `color(srgb 1 1 1 / alpha)`. I imagine this requirement is SDR specific though. Granted, the CSS spec does not currently provide any advice related to handling HDR color spaces (which this library does have) with the algorithm. Maybe this project would prefer to treat all color spaces as SDR with the "CSS" gamut mapping algorithm? I'm not sure. I'm also not aware of an API call to determine if a space is SDR or HDR, but I imagine one could be added if this distinction is desired.

Reply:
From [CSS Color 4](https://drafts.csswg.org/css-color-4/#css-gamut-mapping):

> The CSS gamut mapping algorithm applies to individual, Standard Dynamic Range (SDR) CSS colors which are out of gamut of an RGB display and thus require to be css gamut mapped.

So yes it is explicitly for SDR inputs and returns an SDR RGB color. Which may be an issue for changing the default. On the other hand the old default of reduction in CIE LCH chroma was not especially useful for HDR inputs either.

Confirming that "{ 1 1 1 origin.alpha } in destination" means SDR (media) white.